resources/scripts/ttx_diff.py

```diff
@@ -362,6 +362,22 @@ def sort_fontmake_feature_lookups(ttx):
         for (i, lookup_index) in enumerate(has_value):
             lookup_index.attrib['value'] = str(values[i])
 
+LOOKUPS_TO_SKIP = set([2, 4, 5, 6]) # pairpos, markbase, marklig, markmark
+
+def remove_mark_and_kern_lookups(ttx):
+    gpos = ttx.find("GPOS")
+    if gpos is None:
+        return
+    for lookup in gpos.xpath("//Lookup"):
+        lookup_type = int(lookup.find("LookupType").attrib["value"])
+        if lookup_type not in LOOKUPS_TO_SKIP:
+            continue
+        # remove all the elements but the type:
```

Comment:
explain why it matters to leave the type behind? - my naive thought is it wouldn't

Reply:
if you did remove the type and you then had an empty lookup could you delete the entire lookup?

Reply:
That was my first thought, but if we remove the lookups completely we're leaving invalid ttx, since there will be lookup ids that reference lookups that don't exist. We do expect to always create the same number and type of lookups, so leaving that information in place seemed like a reasonable compromise.